tests/unit/initializers/responsive-test.js

```diff
@@ -1,7 +1,6 @@
 /* global sinon */
 import Ember from 'ember';
 import initializer from '../../../initializers/responsive';
-import instanceInitializer from '../../../instance-initializers/responsive';
```

Comment:
It wasn't being used anywhere in the file... Jshint threw an error.

On Fri, Sep 25, 2015, 9:05 AM Alexandre Baizeau <notifications@github.com>
wrote:

> In tests/unit/initializers/responsive-test.js
> <https://github.com/freshbooks/ember-responsive/pull/41#discussion_r40446193>
> :
>
> > @@ -1,7 +1,6 @@
> >  /* global sinon */
> >  import Ember from 'ember';
> >  import initializer from '../../../initializers/responsive';
> > -import instanceInitializer from '../../../instance-initializers/responsive';
>
> Why is that not necessary anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freshbooks/ember-responsive/pull/41/files#r40446193>.
>